### PR TITLE
(GH-80) Add AutoUpgrade for cChocoPackageInstallerSet

### PR DESCRIPTION
**Pull request by JustinGrote**
Allow for auto-upgrade to be used with installerset as well

### CODE REVIEW

**Comment by pauby** [comment]
@JustinGrote Is there an issue related to this? If not can you raise one?

**Comment by JustinGrote** [comment]
Not sure if I did an issue, I'll prepare.

**Comment by jrdnr** [comment]
@pauby This PR addresses #80

File: DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1 [hunk 1]
```diff
@@ -33,13 +33,18 @@ Composite DSC Resource allowing you to specify multiple choco packages in a sing
         [ValidateNotNullOrEmpty()]
         [System.String]
         $Source
+        [parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
```
Comment (pauby) [inline DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1:37]: A boolean can only have $true or $false. If you pass anything empty it won't convert and will throw an (conversion) exception. So this is effectively redundant.

File: DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1 [hunk 2]
```diff
@@ -33,13 +33,18 @@ Composite DSC Resource allowing you to specify multiple choco packages in a sing
         [ValidateNotNullOrEmpty()]
         [System.String]
         $Source
+        [parameter(Mandatory = $false)]
+        [ValidateNotNullOrEmpty()]
+        [bool]
```
Comment (pauby) [inline DSCResources/cChocoPackageInstallerSet/cChocoPackageInstallerSet.schema.psm1:38]: To keep with the rest of the data types in the parameter declaration can you use [System.Boolean]?

**Review by pauby — CHANGES_REQUESTED** [review]
This is a relatively straghtforward change. Once the minor issues are addressed I'm happy to merge.

**Comment by pauby** [comment]
As the original source repo for this is now gone I'm going to go ahead and close it.

**Pull request closed without merging**